SumoLogic.Logging.Log4Net/SumoLogic.Logging.Log4Net.csproj

```diff
@@ -4,10 +4,10 @@
         <Description>Log4Net appender which sends logs to the Sumo Logic machine data analytics platform.</Description>
         <Authors>Sumo Logic</Authors>
         <Copyright>Copyright © 2018 Sumo Logic Inc. - All Rights Reserved</Copyright>
-        <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+        <TargetFrameworks>net45;netstandard1.3</TargetFrameworks>

```

Comment:
`netstandard2.0` here?

Reply:
Log4net does not have a netstandard2.0 target. So no need.